File: Web/HouseOfSwords/app/Models/Unit.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Unit extends Model
{
    use HasFactory;

    // table properties
    protected $table = 'unitstats';
    protected $primaryKey = 'UnitID';
    public $timestamps = false;

    protected $fillable = [
        "UnitName",
        "UnitSize",
        "AttackValue",
        "DefenseValue",
        "MobilityValue",
        "TrainingTime",
        "TrainingCostGold",
        "TrainingCostFallen",
        "ResearchCost"
    ];
}

File: Game/Assets/Scripts/Models/Unit.cs
[System.Serializable]
public class Unit
{
    public int UnitID;

    public string UnitName;
    public int UnitSize;

    public int AttackValue;
    public int DefenseValue;
    public int MobilityValue;

    public string TrainingTime;

    public int TrainingCostGold;
    public int TrainingCostFallen;
    public int ResearchCost;
}

[System.Serializable]
public class Units
{
    public Unit[] fetchedUnits;
}